### DISCUSSION TITLE
Alist 社区是否考虑参加2023“开源之夏”活动？

### DISCUSSION THREAD

**Discussion by Yefori-Go** [AlistGo/alist#3281] [General]
> 开源之夏是由中科院软件研究所“开源软件供应链点亮计划”发起并长期支持的一项暑期开源活动，旨在鼓励在校学生积极参与开源软件的开发维护，促进优秀开源软件社区的蓬勃发展，培养和发掘更多优秀的开发者。
> 
> 活动联合国内外各大开源社区，针对重要开源软件的开发与维护提供项目任务，并面向全球高校学生开放报名。

参加活动，主办方会为社区提供一部分资金支持，也可以促进 Alist 一些小功能的开发。

[开源之夏官网](https://summer-ospp.ac.cn/)

[社区参与指南](https://summer-ospp.ac.cn/help/community/)

[2022年参与社区](https://summer-ospp.ac.cn/#/org/orglist)

**Comment by oneisall8955** [comment]
虽然但是，该项目薅羊毛特征十分明显

**Reply by RuntimeBroker** [reply]
> 虽然但是，该项目薅羊毛特征十分明显

笑死了

**Reply by Yefori-Go** [reply]
哈哈哈哈哈，假装是分布式存储系统
